app/src/main/res/values-nl/strings.xml

```diff
@@ -40,27 +40,27 @@
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- App title has to have less than 50 characters. If you cannot squeeze it, just remove a part of it -->
-    <string name="app_title">Simple Camera - Capture photos &amp; videos easily</string>
+    <string name="app_title">Eenvoudige Camera - Maak supersnel foto\'s en video\'s</string>
```

Comment:
this is too long, takes 52 chars even without the slashes. Max is 50. Ill remove the ending, adjust it if needed